### PR TITLE
Feature/design

### PR DESCRIPTION
**Pull request by ender554**
Just a good break point with some of the design done. Wanted a check in pull before I start diving deeper into the other style aspects.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -5,7 +5,11 @@ import LoginPage from './components/auth-components/login-page';
 import Lists from './components/Lists';
 import Items from './components/Items';
 import CreateShoppingList from './components/CreateShoppingList';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faIgloo } from '@fortawesome/free-solid-svg-icons';
 
+library.add(faIgloo);
```
Comment (aseehra) [inline src/App.js]: Was this just a test to see that FontAwesomeIcon was working? I can't see any other usage of `FontAwesomeIcon` in the PR?

File: src/components/shoppingLists.js [hunk 2]
```diff
@@ -25,13 +26,14 @@ export class ShoppingLists extends Component {
             onClick={() => this.clickedAList(list.name)}
             to={`/lists/${list.id}`}
           >
-            Name: {list.name} Address: {list.address}
+            {list.name}
```
Comment (aseehra) [inline src/components/shoppingLists.js]: Instead of a <br/> I would wrap the two of these in divs?

File: src/components/shoppingLists.js [hunk 3]
```diff
@@ -25,13 +26,14 @@ export class ShoppingLists extends Component {
...
+            <br /> {list.address}
```
Comment (aseehra) [inline src/components/shoppingLists.js]: I think this should be `list.store.address`

File: src/App.js (hunk 1 above)
Comment (ender554) [reply src/App.js]: deleted

File: src/components/shoppingLists.js (hunk 2 above)
Comment (ender554) [reply src/components/shoppingLists.js]: good call checking now

File: src/components/shoppingLists.js (hunk 3 above)
Comment (ender554) [reply src/components/shoppingLists.js]: that is correct and fixed

**Pull request merged**